### PR TITLE
Chore: Added Linter and Autofomatter

### LINKED ISSUE #4: Add formater to improve code formatting
Investigate libraries to format python code to improve readability and enforce a coding  standard

### PR DESCRIPTION
**Pull request by lucasbotteri**
- Added [Black](https://github.com/psf/black) to auto format code
- Added [Flake8](https://flake8.pycqa.org/en/latest/) to add code linting
- Added recommended vscode setttings
- Run formatter on code
- Updated Readme

### CODE REVIEW

File: back/requirements.txt [hunk 1]
```diff
@@ -1,9 +1,92 @@
+appdirs==1.4.4
```
Comment (BraianMilocco) [inline back/requirements.txt]: Shouldn't exist so many dependences more than "django, restframework, and a couple more"

File: README.md [hunk 2]
```diff
@@ -106,15 +120,14 @@ Optional Description
...
+Please title PR as
 
 `type_of_branch: Main Description`
 
-Add optional Message to the body of the PR 
+Add optional Message to the body of the PR
 
 Once approved please squash PR, title and message should follow same convention as PR.
 
-
```
Comment (BraianMilocco) [inline README.md]: It doesn't appear here:
-----------
## Docs

### Model

[Updated draw io Model](https://drive.google.com/file/d/10hqU88j-wviGUm4WkoZvQdNqPKNEO4dV/view?usp=sharing)

Model

-------------
and in the merge request exist some conflicts in this line with the last merge request

**Review by BraianMilocco — COMMENTED** [review]
requirements.txt is the biggest issue, maybe was done not in the venv and takes system dependences

File: back/requirements.txt (hunk 1 above)
Comment (lucasbotteri) [reply back/requirements.txt]: Fixed !

**Comment by lucasbotteri** [comment]
Branch now updated with develop. Ran black and flake8 again so there're a couple of new changes on the new develop files

**Comment by lucasbotteri** [comment]
@BraianMilocco i suggest you to try this branch. Install new dependencies and follow Readme instructions to use the recommended visual studio configuration, and play around a little bit with the code. 
Try to define unused variables or params inside function to check if flake8 is working correctly. 
Try saving code with a bad format , or strings with `'` instead of `"`. When you save the file, should change it automatically

**Pull request merged**